### ISSUE TITLE
Is there any way to select all filtered rows?

### ISSUE THREAD

**Issue by Michael77** [akveo/ng2-smart-table#406]
I am only able to use the select all button at the top left, but that only selects all rows on the current page.

If I increase the page size to something very large to show all rows that would work, but there are so many rows that it takes too long to display.

**Comment by RoryGarner92** [comment]
You can use the " selectMode: 'multi'," in the settings section.
This will create a checkbox in the header of the of you that will allow you to select all items on that page.

**Comment by Michael77** [comment]
Yes, but I am wondering if it is possible to select all rows on all pages?

**Comment by RoryGarner92** [comment]
Not that I have found.
The only way I found was to set the page rows to the number of rows.
